llrt_utils/build.rs

```diff
@@ -0,0 +1,12 @@
+// WARN: Do not modify this function without changing the other build.rs files in the other crates.
+fn set_nightly_cfg() {
```

Comment:
We can move this to a shared file or create and reuse.

Reply:
Sure